services/net/fileupload/FileUploadService .cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using TNO.Ches;
+using TNO.Ches.Configuration;
+using TNO.Services;
+using TNO.Services.Managers;
+using TNO.Services.FileUpload.Config;
+using Microsoft.Extensions.DependencyInjection;
+
+using TNO.Services.Runners;
+
+namespace TNO.Services.FileUpload;
+
+/// <summary>
+/// FileUploadService class, provides a service to upload files.
+/// </summary>
+public class FileUploadService : IngestService
```

Comment:
Is there a reason it inherits from the IngestService?

Reply:
A lot of the configuration I am referring to other services so I used IngestService. I don't have to connect to Kafka, can I use basicService or create a new class.

Reply:
How does this new service work?  Is it going to be running 24x7?  The services we've built inherit common code because they run 24x7 and have an api attached to them.

Reply:
Since I'm going to use the API in there, I'd like to use CronJob to create a run every night and then close it. Is this ok?